server/pbenchinacan/run-pbench-in-a-can

```diff
@@ -65,6 +75,24 @@ mkdir -p -m 0755  \
     ${SRV_PBENCH}/tmp \
     ${SRV_PBENCH}/cache
 
+cp ${TMP_DIR}/dependency.crt ${TMP_DIR}/elastic.crt
+cp ${TMP_DIR}/dependency.key ${TMP_DIR}/elastic.key
+
+chmod 0644 ${TMP_DIR}/elastic.crt ${TMP_DIR}/elastic.key
```

Comment:
The two distinct patterns of mode bits are disturbing. If they're necessary, a comment would be a good idea...